KILTprotocol/opendid
# feat: add option to check claims using a regex

This adds the option to check claims by regex. It is targeting the openid-connect branch so its easier to review

* adds optional config within the credentialRequirements
* evaluates it within the post-credential handler
* adds documentation about it to the readme

## How to test:

* spin up local setup as described in readme
* add a regex check to the default skyc email credential requirement
* restart the server and check if you can login

## Checklist:

    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here
